frontends/web/src/common/Annotation/ValidateInterface.js

```diff
@@ -95,7 +95,11 @@ class ValidateInterface extends React.Component {
                 .then((result) => {
                   this.setState({ admin_or_owner: result.admin_or_owner });
                 });
-              this.state.task.selected_round = this.state.task.cur_round;
+              this.setState((prevState) => {
+                let task = Object.assign({}, prevState.task);
```

Comment:
Not a major thing, but I think you can mutate state inline with something like:
```
this.setState((prevState) => {
    task : {
        ...prevState.task,
        selected_round: prevState.task.cur_round
    }
})
```
It seems a bit cleaner to me, but ultimately up to you

Reply:
(this comment would apply to everywhere you mutated state)

Reply:
yup, that is indeed a cleaner solution, I'll switch to using that instead.